README.md

```diff
@@ -116,12 +117,19 @@ cd ./api && docker-compose up
 ```
 Start the backend application
 ```
-./gradlew runDockerDb
+./gradlew bootRunDockerDb
 ```
 Run the end to end tests
 ```
-cd ./ui && yarn e2e
+cd ./ui
+yarn cypress
```

Comment:
That is intentional because the workflow could sometimes be such that the servers are already running.  However, if you feel that it would be beneficial to have a cypress-ci command that handled starting the servers running the test and stopping the servers, I could make that change.  Although, I would prefer to do it as part of a smaller stand-alone PR such that we can get this one merged